### ISSUE TITLE
[ICC] If server (raid or full) crashes on Dreamwalker, encounter can't be restarted

### ISSUE THREAD

**Issue by AgeAgainst** [dalaranwow/dalaran-wow#4861]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)

**Expected behaviour**: Tell us what should happen instead

**Steps to reproduce the problem**:

1. Engage Dreamwalker
2. Server DCs
3. Risen Archmages don't respawn.  No way to start the encounter. Prevents a kill of Sindragosa and Lich King

**Include proofs for this behaviour**
http://i.imgur.com/l74saxJ.jpg

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**
http://www.wowhead.com/npc=37868/risen-archmage
Those are the NPCs that are supposed to respawn

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

Unlike the Hodir encounter, you can't do a soft reset on the instance to respawn the mobs.  Waiting 24 hours doesn't do anything.  You're prevented from getting the loot from Dreamwalker, Sindragosa and Lich King.  This is something that has been broken for months.  The last time this happened (over a month ago), we e-mailed community support about the issue.  I had thought someone had opened a ticket but I guess not.  Since it's now happened a second time in our 25 man raid (and 3 times in 10 man) and it's happened in other guild's raids, I'm opening a ticket though kind of surprised one hasn't been opened in the past.
